imgui-SFML.cpp

```diff
@@ -733,7 +736,8 @@ void RenderDrawLists(ImDrawData* draw_data)
             if (pcmd->UserCallback) {
                 pcmd->UserCallback(cmd_list, pcmd);
             } else {
-                GLuint tex_id = (GLuint)*((unsigned int*)&pcmd->TextureId);
+                GLuint tex_id;
+                std::memcpy(&tex_id, &pcmd->TextureId, sizeof(GLuint));
```

Comment:
And here we'll need `GLuint convertToGLTextureHandle(ImTextureID)` to make this easier for future readings of code :)